R/download_ts.R

```diff
@@ -27,7 +27,7 @@
 #'   on_local_exists="skip")
 #' }
 #' @export
-download_ts <- function(var_src, site_name, folder = tempdir(), version=c('tsv','RData'),
+download_ts <- function(var_src, site_name, folder = tempdir(), version=c('tsv','rds','RData'),
```

Comment:
i think there are actually no RData tses - that was just a placeholder for me. want to take it out so it's more clearly just 'tsv' and 'rds' that are relevant?